CMakeLists.txt

```diff
@@ -784,11 +784,17 @@ if (CMAKE_Swift_COMPILER)
 else()
   message(STATUS "Swift Compiler (None).")
 endif()
+
+set(THREADS_PREFER_PTHREAD_FLAG YES)
+include(FindThreads)
```

Comment:
Could you explain what this is for?

Reply:
This is required as CMark builds with threading support, and that requires `Threads::Threads` to be defined to abstract over pthreads vs Win32 vs solaris threads vs HP pthreads. This standard include will ensure that the target is defined.

Reply:
OK, I'm not sure I understand entirely. But I just asked here for future reference when someone wondered why this is here. Adding comment would be great though.